python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -947,11 +947,11 @@ def __defaultActiveBackendsFunc(self):
                 j = stripProxy(self.registry_slice(i))
 
                 job_status = lazyLoadJobStatus(j)
-                backend_obj = lazyLoadJobBackend(j)
-                backend_name = getName(backend_obj)
 
                 if job_status in ['submitted', 'running'] or (j.master and (job_status in ['submitting'])):
                     if self.enabled is True and self.alive is True:
+                        backend_obj = lazyLoadJobBackend(j)
```

Comment:
I guess it can wait until another PR to address this as it's a relatively minimal overhead, unless there are many master jobs in a running state, does hammercloud get into this state?